src/main/java/net/sourceforge/argparse4j/inf/ArgumentParserException.java

```diff
@@ -23,6 +23,9 @@
  */
 package net.sourceforge.argparse4j.inf;
 
+import static net.sourceforge.argparse4j.internal.MessageLocalization.localizeIfPossible;
```

Comment:
In my opinion, inf package should not depend on internal package.  I mean that concrete package depends on abstract package, but not the other way around.  But there are already some cycles here and there.  So I'll do some experiments moving classes around packages to clean up dependencies after merging this PR.

Perhaps, it is my mistake to put ArgumentParserException class in inf package.  It should have been interface.